### PR TITLE
fix: tls hostNameOverride

### PR DESCRIPTION
**Pull request by aleksasiriski**
Previously if you specified `.ingress.tls.hostNameOverride` the deployment of Helm chart fails with:

```
Error: failed to parse /config/values-01_HelmChart.yaml: error converting YAML to JSON: yaml: line 14: did not find expected alphabetic or numeric character
```

### CODE REVIEW

**Comment by james-d-elliott** [comment]
What does this fix?

**Comment by aleksasiriski** [comment]
I've added that to desc of the PR

File: charts/authelia/values.yaml [hunk 1]
```diff
@@ -118,8 +118,7 @@ ingress:
   tls:
     enabled: true
     secret: authelia-tls
-
-    # hostNameOverride:
+    hostNameOverride: ""
```
Comment (james-d-elliott) [inline charts/authelia/values.yaml:121]:
```suggestion
    hostNameOverride: ''
```

**Review by james-d-elliott — CHANGES_REQUESTED** [review]

**Comment by james-d-elliott** [comment]
Taken over locally in #230, thanks!

**Pull request closed without merging**